resources/wikia/libraries/jquery/getResources/jquery.wikia.getResources.js

```diff
@@ -1,101 +1,110 @@
-(function( window, $ ) {
-
-/**
- * Fetches a list of resources and fires a callback when they have all finished
- * loading. Supports CSS, JavaScript, Sass and AssetManager groups.
- *
- * TODO: Right now, asset group file type is determined by searching for a
- * file type extension in the URI. Is there a better way to detect file type?
- *
- * @author macbre
- * @author kflorence
- */
-var getResources = (function() {
-	var rAssetManagerGroup = new RegExp( window.wgAssetsManagerQuery
-			// Escape any special regex characters in the URL
-			.replace( /[\-\[\]{}()*+?.,\\\^$|#\s]/g, '\\$&' )
-			// Escape forward slashes (required for string)
-			.replace( /\//g, '\\/' )
-			// Replace the first parameter to match group or groups
-			.replace( '%1\\$s', 'groups?' )
-			// Replace remaining parameters with wildcard matches
-			.replace( /%\d\\\$(?:s|d)/g, '(.*)' )
-		),
-		rAssetManagerGroupType = /(js|s?css)/,
-		rExtension = /\.(js|s?css)$/;
-
-	return function( resources, success, failure ) {
-		var l, n, matches, remaining,
-			dfd = new $.Deferred();
-
-		// This will be called everytime a resource is loaded
-		var complete = function() {
-			remaining--;
-
-			// All files have been downloaded
-			if ( remaining < 1 ) {
-				if ( typeof success == 'function' ) {
-					success();
+(function (window, $) {
+	'use strict';
+
+	/**
+	 * Fetches a list of resources and fires a callback when they have all finished
+	 * loading. Supports CSS, JavaScript, Sass and AssetManager groups.
+	 *
+	 * This is not recommended for cross-domain JS script loading, as the failure function will not be called.
+	 * @see http://bugs.jquery.com/ticket/13735
+	 *
+	 * TODO: Right now, asset group file type is determined by searching for a
+	 * file type extension in the URI. Is there a better way to detect file type?
+	 *
+	 * @author macbre
+	 * @author kflorence
+	 */
+	var getResources = (function () {
+		var rAssetManagerGroup = new RegExp(window.wgAssetsManagerQuery
+					// Escape any special regex characters in the URL
```

Comment:
these should actually only be indented by 1 tab, but IDE's don't seem to comprehend that.